src/Http/AbstractHttpMessage.php

```diff
@@ -51,11 +53,12 @@ public function addHeader(string $name, string $value)
 
     /**
      * @param string $header
+     * @param mixed  $default
      * @return string
      */
-    public function getHeader($header)
+    public function getHeader(string $header, $default = null)
```

Comment:
@Nek- haha, yeah my comment is missing some words 😅 
Actually, it could be whether a string or `null` right? So the `mixed` type is too generic.
In any case, the return type is wrong as it could be `null` 🙂